### PR TITLE
for editing

### PR DESCRIPTION
**Pull request by ritazakhodyaeva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +9,24 @@
...
+This example shows how use the client-side [ASPxClientDashboard.ItemClick](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboard#js_ASPxClientDashboard_ItemClick) event to get data from a clicked dashboard item.
+<!-- default file list -->
+## Files to Look at
+* [Default.aspx](./CS/WebApp/Default.aspx) (VB:[Default.aspx](./VB/WebApp/Default.aspx))
+* [Default.aspx.cs](./CS/WebApp/Default.aspx.cs) (VB:[Default.aspx.vb](./VB/WebApp/Default.aspx.vb))
+<!-- default file list end -->
+
+## Example Overview
+
+The `ItemClick` event handler receives arguments of the [ASPxClientDashboardItemClickEventArgs](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs) type, which allows you to get data displayed in a clicked item: grid rows, series points, cards, etc. Note that some dashboard items display multidimensional data. See the [Obtaining Underlying and Displayed Data](https://docs.devexpress.com/Dashboard/18078/create-the-designer-and-viewer-applications/web-dashboard/aspnet-web-forms-dashboard-control/obtain-underlying-and-displayed-data) help topic for more information about the general data structure.
```
Comment (dirk-pieterse) [inline README.md]: - Again, `which`
- `etc.` Do not use Latin abbreviations. Both this and **which** are covered in guidelines

File: README.md [hunk 2]
```diff
@@ -10,6 +9,24 @@
...
+
+Follow the steps below to get data from a clicked dashboard item:
+
+1. Call the [GetData](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs#js_ASPxClientDashboardItemClickEventArgs_GetData) method to request the [ASPxClientDashboardItemData](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData) object that contains all information about item data.
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
1. Call the [GetData](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs#js_ASPxClientDashboardItemClickEventArgs_GetData) method to request the [ASPxClientDashboardItemData](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData) object that contains information about the item data.
```

Comment (dirk-pieterse) [reply README.md]: Why did you omit the article here?

File: README.md [hunk 3]
```diff
@@ -10,6 +9,24 @@
...
+2. Call the [ASPxClientDashboardItemData.GetAxisNames](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetAxisNames) method to get a list of displayed axes.
+3. Use the [GetDimensions](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs#js_ASPxClientDashboardItemClickEventArgs_GetDimensions_axisName_) and [GetAxisPoint](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs#js_ASPxClientDashboardItemClickEventArgs_GetAxisPoint_axisName_) methods to get information about dimensions that corresponds to each axis.
```
Comment (dirk-pieterse) [inline README.md]: `dimensions that corresponds` dimension**s** that **correspond**

File: README.md [hunk 4]
```diff
@@ -10,6 +9,24 @@
...
+4. Use the [GetSlice](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetSlice_tuple_) method to divide data by displayed axis points and call the [GetMeasureValue](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetMeasureValue_measureId_) method To get the last level measure values.
```
Comment (dirk-pieterse) [inline README.md]: `divide data by displayed axis points` 'by displayed axis points'? Why **by**? When you divide something **by** something else, it implies you are working with numbers

**Review by dirk-pieterse — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ritazakhodyaeva) [reply README.md]:
```suggestion
The `ItemClick` event handler receives arguments of the [ASPxClientDashboardItemClickEventArgs](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs) type that allows you to get data displayed in a clicked item: grid rows, series points, cards. Note that some dashboard items display multidimensional data. See the [Obtaining Underlying and Displayed Data](https://docs.devexpress.com/Dashboard/18078/create-the-designer-and-viewer-applications/web-dashboard/aspnet-web-forms-dashboard-control/obtain-underlying-and-displayed-data) help topic for more information about the general data structure.
```

File: README.md (hunk 3 above)
Comment (ritazakhodyaeva) [reply README.md]:
```suggestion
3. Use the [GetDimensions](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs#js_ASPxClientDashboardItemClickEventArgs_GetDimensions_axisName_) and [GetAxisPoint](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemClickEventArgs#js_ASPxClientDashboardItemClickEventArgs_GetAxisPoint_axisName_) methods to get information about dimensions that correspond to each axis.
```

File: README.md (hunk 4 above)
Comment (ritazakhodyaeva) [reply README.md]:
```suggestion
4. Use the [GetSlice](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetSlice_tuple_) method to divide data in displayed axis points and call the [GetMeasureValue](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetMeasureValue_measureId_) method to get the last level measure values.
```

Comment (dirk-pieterse) [reply README.md]: - What does it mean `to divide data in displayed axis points`? 
- `the last level measure values` is a noun string that does not make sense

**Review by dirk-pieterse — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (ritazakhodyaeva) [reply README.md]:
```suggestion
4. Use the [GetSlice](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetSlice_tuple_) method to get a data [slice](https://docs.devexpress.com/Dashboard/18078/web-dashboard/aspnet-web-forms-dashboard-control/obtain-underlying-and-displayed-data#slices) that corresponds to the target item's element and call the [GetMeasureValue](https://docs.devexpress.com/Dashboard/js-ASPxClientDashboardItemData#js_ASPxClientDashboardItemData_GetMeasureValue_measureId_) method to get the last level's measure values in the slice.
```

**Pull request merged**